src/webviews/apps/home/components/onboarding.ts

```diff
@@ -50,6 +52,21 @@ export class GlOnboarding extends LitElement {
 		});
 	}
 
+	private renderProgress() {
+		if (!this._state.showWalkthroughProgress) {
+			return null;
+		}
+		return html`
+			<progress
+				class=${classMap({
+					'walkthrough-progress': true,
+					finished: this._state.walkthroughProgress === 1,
+				})}
+				value=${this._state.walkthroughProgress}
+			></progress>
+		`;
+	}
```

Comment:
This progress bar as well as the indicator dot on the walkthrough button feel out of place.

![Screenshot 2024-10-17 at 3 28 44 PM](https://github.com/user-attachments/assets/1e29ea80-5d7f-49ce-be22-db4b225d273b)

@justinrobots - could use some eyes here

Reply:
We should probably use this walkthrough progress UI from new home design. 

![Screenshot 2024-10-18 at 10 51 02 AM](https://github.com/user-attachments/assets/9cf773ed-a675-4865-b38d-8cc09da61482)

See: https://www.figma.com/design/C5tCGmWybTLZJZHmVYJ7Iz/GitLens-Home-Exploration---Q4-2024?node-id=222-9166&node-type=frame&t=g5thrqLApgx7PPuX-0